### PR TITLE
Updated styling

### PR DESCRIPTION
**Pull request by dvcorreia1984**
In this pull request styling was updated according to the following wireframe: 

https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378

1. Missing JSX elements were added. 
2. Styling was updated to match the Wireframe design.
3. Tailwind CSS was also used to do the styling.

### CODE REVIEW

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @dvcorreia1984, 

Good job so far 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 

## Required Changes ♻️

- [x] Please adhere to the project requirements, as per the requirements, your design should match **EXACTLY** same with the **Zeplin** design. I noticed that your design does not cover the whole width of the screen, and the reason behind it you are using a fixed width. Please make the design in such a way that it should always cover the desktop screen.

image

- [x] Please carefully look at the below design comparison and you will find that your design looks unprofessional and inconsistent and it doesn't seem like the original design. Kindly make the changes to make it look exactly like the original design, Have a look carefully at the below comparison. Make sure that your content of navigation should align with the rest of the body element in terms of spacing, padding and width, each part of your project should be the resemeblence of each part of the Zeplin design.

### Your Design
Screenshot from 2023-09-03 02-36-23

### Zeplin Design

Screenshot from 2023-09-03 00-37-49

- [ ] Please fix the path of your `README` license file. Right now your license link is broken and the reason behind it wrong path reference. Kindly add the right path.

Screenshot from 2023-09-03 02-20-56


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dvcorreia1984** [comment]
Thank you for these valuable comments @skyv26. I made improvements to my code based on your recomendations.

File: README.md [hunk 1]
```diff
@@ -153,6 +153,6 @@ Can I contribute to this project?\*\*
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](LICENSE) licensed.
+This project is [MIT](MIT) licensed.
```
Comment (Alusp) [inline README.md]: - [ ] Good job in implementing the changes requested by the previous reviewer, however, as mentioned by him, please ensure that your link is referenced appropriately, The link is still broken and it is showing 404 page again.

404p

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @dvcorreia1984, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descrptive ✔️
- [ ] Professional Github Flow used ✔️
- [ ] Professional PR Summary Flow used ✔️
- [ ] UI Closer to the given design :heavy_check_mark:  

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] You have done extremely well in implementing the changes requested by the previous reviewers, however, there are still some styling you are missing, kindly pay close attention to the given design as requested by the previous reviewers. Matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, Please pay attention to the red markings on your design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

StudentZeps

</td>
      <td>

Zeplin

</td>
    </tr>
  </table>  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @dvcorreia1984 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

:heavy_check_mark: Professional Readme file :+1: 
:heavy_check_mark: Good commit history :+1:
:heavy_check_mark: No linter error :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**